Docs/spring/SpringSecuritySingleSignOnFilter.md

```diff
@@ -171,6 +171,56 @@ The `basicSecurityFilterProvider` bean supports a custom Basic authentication Re
     <property name="realm" value="DemoRealm" />
 </bean>
 ```
+Mixed Single-SignOn and Form
+--------------------------------------
+To support both single sign-on and form-based authentication with spring security similarly to [TomcatMixedSingleSignOnAndFormAuthenticatorValve](https://github.com/Waffle/waffle/blob/master/Docs/tomcat/TomcatMixedSingleSignOnAndFormAuthenticatorValve.md)
```

Comment:
Make this a relative link (remove github.com/....).

Reply:
Sentences need a period at the end.

Reply:
I used an absolute link to be consistent as all links in Docs/spring including this one are already using absolute links

Reply:
Feel free to fix the others too :)